app/src/main/java/ju/video/player/controller/componentscontrollers/ButtonController.java

```diff
@@ -128,8 +139,9 @@ public void verify() {
                         }
                     }
                 }
-                // grab the list of valid videos and set it to the class of video list valid videos
-                //ListValidVideos.getInstance().setVideoList(listvalidVideos);
+                // grab the list of valid videos and set it to the class of video list valid
+                // videos
+                // ListValidVideos.getInstance().setVideoList(listvalidVideos);
             } catch (IOException ioException) {
```

Comment:
Is it ok that the excepcion are a IOException? Shouldn't it be our own exception?